server/schemas/item.graphql

```diff
@@ -67,6 +69,13 @@ extend type Query {
     after: Cursor
     before: Cursor
   ): ItemConnection!
+  itemsByProject(
+    projectId: ID!
+    first: Int
+    last: Int
+    after: Cursor
+    before: Cursor
+  ): ItemConnection!
```

Comment:
```suggestion
```

It is not very useful to get all items across the schema directly from the project.

Reply:
@rot1024 during the discussion with Baba-chan he said that it might be needed in future
however I'll remove it for now